### ISSUE TITLE
invalid literal for int() with base 10

### ISSUE THREAD

**Issue by jeroenterheerdt** [music-assistant/hass-music-assistant#719]
### What version of Music Assistant has the issue?

2022.7.0

### The problem

```
Logger: music_assistant.music.file
Source: /usr/local/lib/python3.10/site-packages/music_assistant/music_providers/filesystem.py:247
First occurred: 5:09:33 PM (4 occurrences)
Last logged: 5:09:38 PM

Error processing /media/albums/3 Doors Down/Kryptonite/01 - Kryptonite.mp3 - invalid literal for int() with base 10: ''
Error processing /media/albums/4hero/Play with the Changes/01 Morning Child.mp3 - invalid literal for int() with base 10: ''
Error processing /media/albums/4hero/Play with the Changes/14 Gonna Give It Up (Wanna Quit).mp3 - invalid literal for int() with base 10: ''
Error processing /media/albums/Aaliyah/Are You That Somebody/02 - Are U That Somebody.mp3 - invalid literal for int() with base 10: ''
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/music_assistant/music_providers/filesystem.py", line 234, in sync_library
    track = await self._parse_track(entry.path)
  File "/usr/local/lib/python3.10/site-packages/music_assistant/music_providers/filesystem.py", line 544, in _parse_track
    track.disc_number = tags.disc
  File "/usr/local/lib/python3.10/site-packages/music_assistant/helpers/tags.py", line 97, in disc
    return int(tag.split("/")[0])
ValueError: invalid literal for int() with base 10: ''

```

### How to reproduce

have mp3 on file share.

### Relevant log output

see above.

### What version of Home Assistant Core are your running

2022.7.3

### What type of installation are you running?

Home Assistant Core

### On what type of hardware are you running?

Raspberry Pi

**Comment by OzGav** [comment]
Marcel will have a better idea but have you tagged your files properly? The error relates to some issue with the tags. We recommend using Musicbrainz Picard to minimise these sorts of issues.

**Comment by marcelveldt** [comment]
Tagging issue indeed. Looking at the error there's a string inside the disc field that can't be parsed to a number.

Please keep this issue open so I can throw a bit more specific exception here

**Comment by jeroenterheerdt** [comment]
my tag sources come from Musicbrainz as well as other sources. Discnumber is somethings numeric, but often a string: 1/2 or 2/2 for example. I guess that's what the problem is.

**Comment by OzGav** [comment]
The “other sources” is probably the issue. If you can tag all of your music exclusively with Picard that will probably fix these issues for you.

**Comment by jeroenterheerdt** [comment]
I beg to differ. the ID3V2v2.3.0 standard is very clear: the TPOS attribute can absolutely contain a '/` to indicate a double CD. See: https://id3.org/id3v2.3.0, search for TPOS:  `TPOS
The 'Part of a set' frame is a numeric string that describes which part of a set the audio came from. This frame is used if the source described in the "TALB" frame is divided into several mediums, e.g. a double CD. The value may be extended with a "/" character and a numeric string containing the total number of parts in the set. E.g. "1/2".`

So it's great that Musicbrainz chooses to just only do numerical discnumber, but other sources don't and supply 1/2 or 2/2, which is perfectly fine following the standard. So IMHO music assistant should not break on a TPOS tag that contains a `/`.

**Comment by marcelveldt** [comment]
No worries, it didn't break on the slash but on the empty string. It's fixed in 2022.7.1 patch release.
Thanks for reporting. If you still experience any issues after installing the 2022.7.1 patch let me know!
